src/arch/armv8/armv8-a/psci.c

```diff
@@ -56,17 +56,17 @@ static void psci_wake_from_powerdown(void)
     vcpu_run(cpu()->vcpu);
 }
 
-static void psci_wake_from_idle(void)
+static void psci_wake_from_standby(void)
 {
-    cpu_idle_wakeup();
+    cpu_standby_wakeup();
 }
 
 void psci_wake_from_off(void);
 
 void (*psci_wake_handlers[PSCI_WAKEUP_NUM])(void) = {
     [PSCI_WAKEUP_CPU_OFF] = psci_wake_from_off,
     [PSCI_WAKEUP_POWERDOWN] = psci_wake_from_powerdown,
-    [PSCI_WAKEUP_IDLE] = psci_wake_from_idle,
+    [PSCI_WAKEUP_IDLE] = psci_wake_from_standby,
```

Comment:
> PSCI_WAKEUP_IDLE

Shouldn't this be renamed to standby or maybe suspend?

Reply:
I think João changed this already.